.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -4,13 +4,19 @@ about: Create a report for a documentation problem.
 labels: A-docs
 ---
 <!--
+
 Thank you for finding a documentation problem! 📚
 
 Documentation problems might be grammatical issues, typos, or unclear wording, please provide details regarding the documentation including where it is present.
 
+Note that this issue tracker/template is for std/compiler docs, the unstable book, and some more docs.
+If you find an issue related to the docs listed in https://github.com/rust-lang/rust/tree/master/src/doc as Git submodules,
```

Comment:
Can this just list those issue trackers? I don't suspect most people know how to deal with submodules.

Maybe something like:

> Note: If your issue is for one of these, please use their dedicated issue tracker instead:
>
> * The Rust Book: https://github.com/rust-lang/book/issues
> * Rust by Example: https://github.com/rust-lang/rust-by-example/issues
> * The Edition Guide: https://github.com/rust-lang/edition-guide/issues
> * The Cargo Book: https://github.com/rust-lang/cargo/issues
> * The Clippy Book: https://github.com/rust-lang/rust-clippy/issues
> * The Reference: https://github.com/rust-lang/reference/issues
> * The Rustonomicon: https://github.com/rust-lang/nomicon/issues
> * The Embedded Book: https://github.com/rust-embedded/book/issues
>
> All other documentation issues should be filed here.

Reply:
Thanks for the list! I was initially worried that such a list would get outdated, but on second thought, it rarely happens and the list would be helpful in most cases.